src/EPR.RegulatorService.Facade.API/Controllers/RegulatorController.cs

```diff
@@ -115,7 +122,8 @@ public async Task<IActionResult> UpdateEnrolment([FromBody]UpdateEnrolmentReques
         }
         catch (Exception e)
         {
-            _logger.LogError(e, $"Error updating the enrolment {updateEnrolmentRequest.EnrolmentId} by the user {User.UserId()}");
```

Comment:
The standard way to use Microsoft logger in other project is as follows
 _logger.LogError("Error updating the enrolment {EnrolmentId} by the user {UserId}", updateEnrolmentRequest.EnrolmentId, User.UserId());